modules/kubernetes-configuration/flux-configuration/main.bicep

```
@@ -63,7 +63,7 @@ resource managedCluster 'Microsoft.ContainerService/managedClusters@2022-07-01'
   name: clusterName
 }
 
-resource fluxConfiguration 'Microsoft.KubernetesConfiguration/fluxConfigurations@2022-03-01' = {
+resource fluxConfiguration 'Microsoft.KubernetesConfiguration/fluxConfigurations@2023-05-01' = {
```

```suggestion
resource fluxConfiguration 'Microsoft.KubernetesConfiguration/fluxConfigurations@2022-11-01' = {
```

@aadev1, I successfully tested this API version which is newer than the old one, but not as new as the one you proposed. 
Note: This change would require you to regenerate the readme - which I guess you'd need to do anyways using the `utilities/tools/Set-ModuleReadMe.ps1` utility - and compile the JSON template using the `bicep build '<bicepPath>'` utility.

---

Ok nevermind, different plan

---

Using Api version 2023-05-01 need at least one kustomizations. If you switch to the newest version please set this parameter to Required.